docs/release_notes/release_notes.md

```diff
@@ -11,8 +11,67 @@ has_children: true
 
 ## Version 4.9.0 (July 2024)
 
+* Added support for Nessus:Pro (Nessus Professional). Check the upgrade guide to onboard the logs.
+
+* Added checkbox on alert page to prevent the alert state from being updated from the Cyences setup page. Check the upgrade guide for more details.
+
+* Added the following panels for user login activity on the **Windows** dashboard:
+    * Successful Login Events
+    * Failed Login Attempts by User
+
+* Added the following panels to the **Vulnerability** dahboard:
+    * Total Vulnerability Count By Severity
+    * New Total Vulnerability Found Over Time
+
+* Added a separate alert for password expired events called **AD - Login Failure due to Password Expired** and filtered out the password expired events from authentication alerts.
+
+* Added a section to configure the digest email configs on **Cyences Settings > Cyences App Configuration > Cyences Email Action Configuration**.
+
+* Removed the following alerts as it contains static lookup which is not maintainable and causes false positives:
+    * Email - Suspicious Subject or Attachment
+    * Email - With Known Abuse Web Service Link
+    * Ransomware - Endpoint Compromise - Malicious Package Found
+
+
+* ### Enhancements
+
+    * #### Sophos Endpoint data
+        * Removed the Sophos UI configuration from Cyences configuration page & sophosinstancedetails custom command to fetch sophos endpoints.
+        * Sophos device inventory alert will now support the 'sophos_endpoints' source. Check upgrade guide to collect the sophos endpoint logs.
```

Comment:
Do not reference the upgrade guide, Instead mention the onboarding information directly.